pokedex/pokedex.lua

```diff
@@ -394,9 +436,31 @@ function M.get_pokemon(pokemon)
 	end
 end
 
+function M.get_pokemon(pokemon, variant)
+	local raw = M.get_pokemon_raw(pokemon)
```

Comment:
Would it be cleaner to make the get_pokemon_raw into a local function? 

```
local function get_pokemon(pokemon)
  -- current M.get_pokemon_raw
end

function M.get_pokemon(pokemon, variant)
  local raw = get_pokemon(pokemon)
  if not variant or not raw.Variants or not raw.Variants[variant] then
		return raw
  end
 -- rest of function
end
``` 
That way it will only be one way to get a Pokemon, if you don't want the variant then you pass in `nil` instead of using `_raw`.

Reply:
Yes, for some reason when I made it I thought I had need of the raw data outside of `pokemon.lua`, but I guess I moved away from needing that!